### PR TITLE
Aizad/FEQ-1725/Tailwind-Setup

### PR DESCRIPTION
**Pull request by aizad-deriv**
- Installing `tailwindcss`, `postcss` and `autoprefixer` within the project.
- Setting up `tailwind.config.js` & `postcss.config.js` within the project.
- Utilize values from `themes.scss` for the colors, font sizes, line height and etc.
- supports both light and dark mode based on the current deriv app theme switcher.
- import themes via preset.
- convert themes file in scss to css.

### CODE REVIEW

File: lib/styles/index.scss [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 
+// @import "./shared/reset.scss";
+// @import "./shared/constants.scss";
+// @import "./shared/devices.scss";
+// @import "./shared/fonts.scss";
+// @import "./shared/google-fonts.scss";
+// @import "./shared/themes.scss";
+// @import "./wallet/styles/animations.scss";
+// @import "./wallet/styles/devices.scss";
+// @import "./wallet/styles/fonts.scss";
```
Comment (shayan-deriv) [inline lib/styles/index.scss:13]: can we remove them for now?

Comment (aizad-deriv) [reply lib/styles/index.scss:13]: Sure but I still need to use them as reference 😅 will remove it once I'm done with everything 👍🏼

File: tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
+/** @type {import('tailwindcss').Config} */
+
+const BASE_COLORS = {
```
Comment (likhith-deriv) [inline tailwind.config.js:3]: Will u be adding all [these colors](https://github.com/deriv-com/ui/blob/main/lib/styles/shared/constants.scss) later?

Comment (aizad-deriv) [reply tailwind.config.js]: @likhith-deriv yes I will all of them eventually. But right now I'm just checking if it's working properly.

**Pull request closed without merging**